<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creator de probleme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Titlu:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="titleLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Descriere:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="descriptionLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statementLabel">
      <property name="text">
       <string>Cerinta:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="statementTextEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="testsListLabel">
      <property name="text">
       <string>Teste:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="testsListLayout">
      <item>
       <widget class="QListWidget" name="testsList"/>
      </item>
      <item>
       <widget class="QPushButton" name="addTestButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="testsListButtons"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="pathLabel">
      <property name="text">
       <string>Locatie de salvare:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="pathLayout">
      <item>
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Navigheaza</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="finalButtonsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Anuleaza</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Adauga</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
